app/js/components/shared/CustomDatePicker.jsx

```diff
@@ -0,0 +1,78 @@
+import React, { PureComponent } from 'react';
+import DatePicker from 'react-datepicker';
```

Comment:
It seems like this is generic enough that it could be removed into React-Components?  I would add it to a new directory "src/components/widgets"